baybe/simulation/scenarios.py

```diff
@@ -136,7 +137,14 @@ def unpack_simulation_results(array: DataArray) -> pd.DataFrame:
     # Simulate and unpack
     result_variable = "simulation_result"
     batch_simulator = make_xyzpy_callable(result_variable)
-    da_results = batch_simulator.run_combos(combos)[result_variable]
+
+    with warnings.catch_warnings():
```

Comment:
In the PR description, you said that you tried some stuff that did not happen. Could you very briefly elaborate on what you tried?

Reply:
this relates to the 1st argument of `.filterwarnings`
Besides `ignore` it also accepts `once` or `module`. These would make even more sense that an all-ingore because then it would show only 1 warning for the entire simulation which i guess is much more acceptable than millions of warnings. But they dont seem to work, ie I get all warnings. The internet is full of posts complaining that these options dont work in their case

Reply:
@AVHopp  do you require anything else from me?